workflows/kfdrc-germline-sv-wf.cwl

```diff
@@ -16,39 +116,43 @@ inputs:
   indexed_reference_fasta:
     type: 'File'
     secondaryFiles:
-      - {pattern: '^.dict', required: true}
-      - {pattern: '.fai', required: true}
-      - {pattern: '.64.alt', required: false}
-      - {pattern: '.64.amb', required: true}
-      - {pattern: '.64.ann', required: true}
-      - {pattern: '.64.bwt', required: true}
-      - {pattern: '.64.pac', required: true}
-      - {pattern: '.64.sa', required: true}
+    - {pattern: '^.dict', required: true}
+    - {pattern: '.fai', required: true}
+    - {pattern: '.64.alt', required: false}
+    - {pattern: '.64.amb', required: true}
+    - {pattern: '.64.ann', required: true}
+    - {pattern: '.64.bwt', required: true}
+    - {pattern: '.64.pac', required: true}
+    - {pattern: '.64.sa', required: true}
+    doc: |
+      The reference genome fasta (and associated indicies) to which the germline BAM was aligned.
     "sbg:fileTypes": "FASTA, FA"
-    "sbg:suggestedValue": {
-      class: File, path: 60639014357c3a53540ca7a3, name: Homo_sapiens_assembly38.fasta,
-      secondaryFiles: [
-        {class: File, path: 60639019357c3a53540ca7e7, name: Homo_sapiens_assembly38.dict},
+    "sbg:suggestedValue": {class: File, path: 60639014357c3a53540ca7a3, name: Homo_sapiens_assembly38.fasta,
+      secondaryFiles: [{class: File, path: 60639019357c3a53540ca7e7, name: Homo_sapiens_assembly38.dict},
         {class: File, path: 60639016357c3a53540ca7af, name: Homo_sapiens_assembly38.fasta.fai},
         {class: File, path: 60639019357c3a53540ca7eb, name: Homo_sapiens_assembly38.fasta.64.alt},
         {class: File, path: 6063901f357c3a53540ca84d, name: Homo_sapiens_assembly38.fasta.64.amb},
         {class: File, path: 6063901f357c3a53540ca849, name: Homo_sapiens_assembly38.fasta.64.ann},
         {class: File, path: 6063901d357c3a53540ca81e, name: Homo_sapiens_assembly38.fasta.64.bwt},
         {class: File, path: 6063901c357c3a53540ca801, name: Homo_sapiens_assembly38.fasta.64.pac},
-        {class: File, path: 60639015357c3a53540ca7a9, name: Homo_sapiens_assembly38.fasta.64.sa}
-      ]
-    }
-  input_bams: {type: 'File[]', secondaryFiles: [{pattern: '^.bai', required: false}, {pattern: '.bai', required: false}], doc: "Input BAM file", "sbg:fileTypes": "BAM"} 
+        {class: File, path: 60639015357c3a53540ca7a9, name: Homo_sapiens_assembly38.fasta.64.sa}]}
+  germline_reads: {type: 'File', secondaryFiles: [{pattern: '^.bai', required: false},
```

Comment:
Why not leave as a file array?

Reply:
I did it mostly for batch running I guess. This way the user can batch on germline reads. Also I don't think we want to be using this tool as a joint caller do we?

Reply:
🤷🏻‍♂️ we can change later if needed